worker/jobTypes/githubJob.js

```diff
@@ -314,11 +296,14 @@ class GitHubJobClass {
                 });
             });
         } catch (error) {
+          if (error.code === 1) {
```

Comment:
hmmm if we know this code is broken - do we want to merge it to master? without makefile changes, no errors will be generated. but technically this stanza is unnecessary in this PR

Reply:
Merged DOP-64 so this should be resolved.